doc/project/hw_stages.md

```diff
@@ -181,8 +199,46 @@ The expectation is that this would create its life as a newly tracked revision n
 Possibility 3: if enough new features are requested to greatly change the spirit of the design, a new version increment of 1.0 would be created in a fashion similar to above.
 This would require a new RFC process, and thus the Life Stage would start again as L0 - Specification.
 
+### Multiple versions of a design
+
+In the current state of the project, we have not completely generated the full feature list for the product.
+For launch we consciously chose to take five designs to signoff as mentioned above.
```

Comment:
It feels like these two lines are answering an unasked question. Recommend cutting them.

Reply:
Rephrased, PTAL